ome_zarr/napari.py

```diff
@@ -62,6 +62,24 @@ def f(*args: Any, **kwargs: Any) -> List[LayerData]:
                     layer_type = "labels"
                     if "colormap" in metadata:
                         del metadata["colormap"]
+                    if "properties" in metadata:
```

Comment:
This will always be True since you are always adding `"properties": properties,` in the reader, even if the `properties` is an empty dict. In which case your `reader_props` below will be `{'index': []}`, which doesn't break anything but could be cleaner.

Reply:
I'll change this so we don't add properties if there's none to add